documentation/src/main/asciidoc/entity-view/manual/en_US/11_spring_data.adoc

```diff
@@ -77,7 +83,7 @@ Optionally specify a custom basePackage for repository class scanning and a cust
 [[spring-data-features]]
 === Features
 
-The integration comes with a convenience base interface `com.blazebit.persistence.impl.springdata.repository.EntityViewRepository`
+The integration comes with a convenience base interface `com.blazebit.persistence.impl.springdata.api.repository.EntityViewRepository`
```

Comment:
Please scan through the workspace and try to find usages of ".springdata." and replace the usages with the proper namespace.